fearless/Modules/About/AboutViewController.swift

```diff
@@ -1,236 +1,106 @@
 import UIKit
 import SoraUI
+import CommonWallet
 
-final class AboutViewController: UIViewController, AdaptiveDesignable {
-    private enum Constants {
-        static let logoTopOffset: CGFloat = 102.0
-        static let tableTopOffset: CGFloat = 279
-        static let heightFriction: CGFloat = 0.85
-    }
+final class AboutViewController: UIViewController, AdaptiveDesignable, ViewHolder {
+    typealias RootViewType = AboutViewLayout
 
-    private enum Row {
-        static let height: CGFloat = 48.0
+    // MARK: - Constants
 
-        case website
-        case opensource
-        case social
-        case writeUs
-        case terms
-        case privacy
+    private enum Constants {
+        static let rowHeight: CGFloat = 48.0
     }
 
-    private enum Section: Int, CaseIterable {
-        static let height: CGFloat = 68.0
-
-        case about
-
-        var rows: [Row] {
-            switch self {
-            case .about:
-                return [.website, .opensource, .social, .writeUs, .terms, .privacy]
-            }
-        }
-    }
+    // MARK: - Private properties
 
-    var presenter: AboutPresenterProtocol!
+    private var locale = Locale.current
+    private var presenter: AboutPresenterProtocol
+    private var viewModel: [AboutViewModel]?
```

Comment:
instead of this we always use *ViewState pattern. so you need to create AboutViewState enum, please take a look "WalletSend" module for example